knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -511,35 +510,6 @@ func enableControlPlaneManifest(spec serverlessoperatorv1alpha1.KnativeKafkaSpec
 	return spec.Broker.Enabled || spec.Sink.Enabled || spec.Source.Enabled || spec.Channel.Enabled
 }
 
-func configureLegacyEventingKafka(kafkachannel serverlessoperatorv1alpha1.Channel) mf.Transformer {
```

Comment:
this func is also called/used on the unit tests

Reply:
Ok, I deleted them all and a string search doens't show anything